### DISCUSSION TITLE
:class inside @foreach

### DISCUSSION THREAD

**Discussion by BSN4** [livewire/livewire#7936] [7. Bugs (archived)]
### Livewire version

v3

### Laravel version

v10

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

```php
//outside the loop :class works
Weight chosen: <h2 :class="$wire.option == '3' && 'bg-sky-600'"
                                   class="text-white" x-text="$wire.option"></h2>
        <fieldset class="mt-4">
            <legend class="sr-only">Choose a weight</legend>
            <div class="grid grid-cols-4 gap-4 sm:grid-cols-8 lg:grid-cols-4">
                @foreach($product->options as $option)

// :class won't work here
                    <label :class="$wire.option == '{{ $option->id }}' && 'border-sky-600'"
                           class="group">
                        <input type="radio" name="option" value="{{ $option->id }}" wire:model="option"
                               class="sr-only"
                               aria-labelledby="option-{{$product->id}}-label">
                        <span id="option-{{$product->id}}-label">{{ $option->title }}</span>
                        <span class="pointer-events-none absolute -inset-px rounded-md"
                              aria-hidden="true"></span>
                    </label>
                @endforeach
            </div>
        </fieldset>

```

### Repository or Wirebox URL

https://github.com

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by sfc3lf44** [comment]
Can you share how you resolved this issue?

**Reply by BSN4** [reply]
this code works it was just my styles ("border-sky-600") wasn't working (not in the compiled css) thats why i closed it 

just do 

pnpm run build

or if you don't have pnpm

npm run build

**Reply by BSN4** [reply]
sorry for posting it without confirmation
